add missing Issues content for Review List GA (#3255)
getsentry/sentry-docs: src/docs/product/issues/issue-details/index.mdx

![Issue details](issue-details.png)

While the Issue Details page displays information about a specific error event that's part of an issue, you can navigate between the events of an issue using the "Older" and "Newer" buttons.

In the right hand sidebar, [sentry.io](https://sentry.io) reflects a summary of the that includes information such as how often the issue has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. You can also use this section to link to existing GitHub or Jira issues. Learn more about linking issues in [Integrations](/product/integrations/). Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the Global Header.

In addition, this page provides several other key pieces of information.

## Suspect Commits

> A _suspect commit_ is a commit that's been identified as potentially having caused the error event. If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.

Author: ```suggestion
A _suspect commit_ is a commit that's been identified as potentially having caused the error event. If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The author of the suspect commit is also suggested as an assignee for the issue.
```

## Tags

[Tags](/product/sentry-basics/search/) are key/value string pairs that are both indexed and searchable. For example, a tag provides you with information such as the browser, device, or user associated with the event. Tags are the diagnostic information sent by the SDK for the individual event.

The tags displayed in the main section of this page are specific to the event that you're viewing. In contrast, the tags displayed in the right-hand sidebar are a summary of all tag values for all events included in the issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.

## Exception (Stack Trace)

> The Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on.

Author: ```suggestion
The Exception section of the page displays the stack trace, which shows you the line of code that the event errored on.
```

![Exception](issue-exception.png)

> It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.

Author: ```suggestion
It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue. You can also customize how events are grouped. Learn more about customized grouping in [Grouping Issues](/product/sentry-basics/guides/grouping-and-fingerprints/).
```

## Breadcrumbs

> Breadcrumbs provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript.

Author: ```suggestion
[Breadcrumbs](/product/issues/issue-details/breadcrumbs/) provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript.
```

---
title: Issue Details
sidebar_order: 10
description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
---

> The Issue Details page helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific error event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue.

Author: @adhiraj Robin has dropped in a lightly more generic image that I think is a good way to go for now.
